js/ad_manager_template.js

```diff
@@ -25,6 +25,7 @@ OO.Ads.manager(() => {
 
     let amc = null;
     const remoteModuleJs = 'http://my.company/myAdModule.js';
+    // eslint-disable-next-line no-unused-vars
```

Comment:
it's used on the line 51

Reply:
agreed with @alexeyMohnatkin that this is the best option